minio/helpers.py

```diff
@@ -46,6 +46,7 @@
 # Constants
 MAX_MULTIPART_COUNT = 10000 # 10000 parts
 MAX_MULTIPART_OBJECT_SIZE = 5 * 1024 * 1024 * 1024 * 1024  # 5TiB
+MAX_PART_OBJECT_SIZE = 5 * 1024 * 1024 * 1024  # 5GiB
```

Comment:
It is probably better to call this MAX_PART_SIZE, as MIN_PART_SIZE

Reply:
Done